.github/workflows/publish.yml

```diff
@@ -9,9 +10,39 @@
 name: Upload Python Package
 
 on:
-  release:
+  release
     types: [published]
 
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        ref: ${{ github.event.release.tag_name }}
+    - name: Install Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+        # architecture: 'x64'
+
+    - name: Setup pip cache
+      uses: actions/cache@v3
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-3.10-${{ hashFiles('pyproject.toml') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-3.10-
+    - name: Build extension package
+      run: |
+        python3 -m pip install hatch
+
+        hatch build
+    - name: Publish extension package
+      # if: startsWith(github.ref, 'refs/tags')
+      uses: pypa/gh-action-pypi-publish@release/v1
+      with:
 permissions:
```

Comment:
remove lines marked in this link - https://github.com/huntdatacenter/jupyter-pgadmin-proxy/blob/195bfc23f91db2f99d25ea502bae4ff324e2c4f3/.github/workflows/publish.yml#L46-L68